packages/chargebee/README.md

```diff
@@ -0,0 +1,108 @@
+# `@goodhood/chargebee`
+
+> React chargebee components
+
+## Install
+
+```
+npm i @goodhood/chargebee
+```
+
+Install peer dependencies
+```
+npm i react // v16.x.x
+npm i react-load-script // v0.x.x
+npm i prop-types // v15.x.x
+npm i @babel/runtime // v7.x.x
+```
+
+## Include:
+
+```js
+import { Action, PortalLink, Checkout } from '@goodhood/chargebee';
+import PortalLik from '@goodhood/chargebee/lib/portal_link';
+import Action from '@goodhood/chargebee/lib/action';
+import Checkout from '@goodhood/chargebee/lib/checkout';
+import { SECTION_INVOICES, SECTION_EDIT_SUBSCRIPTION, SECTION_BILLING_ADDRESS } from '@goodhood/chargebee/lib/constants';
+```
+
+# API
+
+## Action
+```jsx
+import { Action } from '@goodhood/chargebee';
+
+const App = () => (
+  <Action
+    {/* Your site name */}
+    site="site-name"
+
+    {/* Will be called on element click. Receives chargebee instance as an argument */}
+    onCall={(chargebeeInstance, ChargebeeGlobalObject) => {}}
```

Comment:
`ChargebeeGlobalObject` vs `chargebeeGlobalObject` ?

Reply:
in window its names as Chargebee

Reply:
it's a singleton class implementation, perhaps ChargebeeSingleton would be clearer?

Reply:
renamed to ChargebeeAPI